src/ros_video_components/include/ros_video_components/ros_battery_indicator.hpp

```diff
@@ -0,0 +1,48 @@
+#ifndef ROS_BATTERY_INDICATOR_HPP
+#define ROS_BATTERY_INDICATOR_HPP
+
+//QT
+#include <QQuickPaintedItem>
+#include <QPainter>
+#include <QObject>
+
+//ROS
+#include <ros/ros.h>
+#include <std_msgs/Float32.h>
+
+
+class  ROS_Battery_Indicator: public QQuickPaintedItem {
```

Comment:
This can probably just extend `QQuickItem` as it doesn't need the functionality of `QQuickPaintedItem`